### ISSUE TITLE
Access to the Indoor dataset and specific Places images for test ( What was your test image with masks)

### ISSUE THREAD

**Issue by kiashann** [DQiaole/ZITS_inpainting#22]
Hi. How we can access to the Indoor dataset for training?
How we can access  to the test set  train set for the Places and Indoor datasets? 
@DQiaole

**Comment by DQiaole** [comment]
Hi~
You can download Indoor dataset from https://pan.baidu.com/s/11O1Q7gcn7dhjPDDUNrmodQ, password:hfok; and we have split the Indoor for training and validation.
As for Places2, we use the training set officially provided by Places2, which has 180w images; and we sampled 100 images randomly from each class in the Places2 test set, resulting in a total of 36,500 images as our test set.

**Comment by kiashann** [comment]
> hfok

We can not download any files from pan.baidu.com when we are not in the China. Could you please share this dataset on google drive?
I need you test set ( images and masks) for the Places and Indoor dataset for comparison. Could you please share with me?

**Comment by DQiaole** [comment]
We have upload the test set ( images and masks) for the Places and Indoor dataset.
If you cannot download the Indoor from pan.baidu.com, we recommend you download ShanghaiTech and NYUDepthV2, and construct the Indoor dataset according to our data split file.

**Comment by kiashann** [comment]
Please upload the Indoor dataset in another Cloud.
In this [file](https://github.com/DQiaole/ZITS_inpainting/blob/main/test_mask.zip) you shared test set for Indoor and places dataset. Isn't it?

**Comment by DQiaole** [comment]
Sorry, our Google Drive has no more space, we can only provide the validation set of indoor through google drive: https://drive.google.com/file/d/1LiJ1b_Cedf_9yQHPcAyDpxgkbcFrtF91/view?usp=sharing.

And this [file](https://github.com/DQiaole/ZITS_inpainting/blob/main/test_mask.zip) is the test masks of our test images.

**Comment by kiashann** [comment]
> Sorry, our Google Drive has no more space, we can only provide the validation set of indoor through google drive: https://drive.google.com/file/d/1LiJ1b_Cedf_9yQHPcAyDpxgkbcFrtF91/view?usp=sharing.
> 
> And this [file](https://github.com/DQiaole/ZITS_inpainting/blob/main/test_mask.zip) is the test masks of our test images.

The test mask file is for Indoor or Places dataset?

**Comment by DQiaole** [comment]
yes, indoor test images and places2 test images use the same test masks.

**Comment by DQiaole** [comment]
Besides, we have uploaded the full indoor dataset, you can find them in README.md.

**Comment by kiashann** [comment]
> Besides, we have uploaded the full indoor dataset, you can find them in README.md.

Thanks
